go-controller/hybrid-overlay/pkg/controller/node_linux_test.go

```diff
@@ -409,12 +453,41 @@ var _ = Describe("Hybrid Overlay Node Linux Operations", func() {
 
 			err = n.controller.EnsureHybridOverlayBridge(node)
 			Expect(err).NotTo(HaveOccurred())
+			linuxNode, okay := n.controller.(*NodeController)
+			Expect(okay).To(BeTrue())
+			Expect(linuxNode.initialized).To(BeTrue())
+			// ovs commands generated by the initial sync
+			fexec.AddFakeCmdsNoOutputNoError([]string{
+				"ovs-ofctl dump-flows --no-stats br-ext table=20",
+				"ovs-ofctl -O OpenFlow13 --bundle replace-flows br-ext -",
+			})
 
-			// FIXME
-			// Eventually(fexec.CalledMatchesExpected, 2).Should(BeTrue(), fexec.ErrorDesc)
+			//perform the requested cacheSync
+			linuxNode.syncFlows()
+			initialFlowCache := map[string]*flowCacheEntry{
+				"0x0": generateInitialFlowCacheEntry(),
+			}
+			Expect(compareFlowCache(linuxNode.flowCache, initialFlowCache)).To(BeTrue())
+
+			Eventually(fexec.CalledMatchesExpected, 2).Should(BeTrue(), fexec.ErrorDesc)
 			validateNetlinkState(thisSubnet, "1.2.3.3")
 			err = n.controller.AddPod(testPod)
+			fmt.Printf("KEYWORD: PODIPTOCOOKIE %s\n", podIPToCookie(net.ParseIP(pod1IP)))
 			Expect(err).NotTo(HaveOccurred())
+			// ovs commands generated by second cacheSync
+			fexec.AddFakeCmdsNoOutputNoError([]string{
+				"ovs-ofctl dump-flows --no-stats br-ext table=20",
+				"ovs-ofctl -O OpenFlow13 --bundle replace-flows br-ext -",
+			})
+
+			//perform the requested cacheSync
+			linuxNode.syncFlows()
+			initialFlowCache[podIPToCookie(net.ParseIP(pod1IP))] = &flowCacheEntry{
+				flows:       []string{"table=10,cookie=0x01020305,priority=100,ip,nw_dst=1.2.3.5,actions=set_field:22:33:44:55:66:77->eth_src,set_field:aa:bb:cc:dd:ee:ff->eth_dst,output:ext"},
```

Comment:
use the constants and not hardcoded strings, also add a comment describing what flow this is checking

Reply:
done